client/lib/emails/has-unused-mailbox-warnings.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Internal dependencies
+ */
+import { EMAIL_WARNING_SLUG_UNUSED_MAILBOXES } from './email-provider-constants';
+import { hasWarningsWithSlug } from './helpers';
+
+export function hasUnusedMailboxWarnings( emailAccount ) {
```

Comment:
Should this not be `hasUnusedMailboxWarning()`? i.e. singular rather than plural, as we'll return true if only one unused mailbox warning exists.